pkgs/applications/misc/fluidd/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenvNoCC, fetchurl, unzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "fluidd";
+  version = "1.16.2";
+
+  src = fetchurl {
+    name = "fluidd-v${version}.zip";
+    url = "https://github.com/cadriel/fluidd/releases/download/v${version}/fluidd.zip";
+    sha256 = "1qwj25xvvxvm1fxx216nn2gp7js4d682mm3l4s7ns90fc5ygvc8i";
+  };
+
+  nativeBuildInputs = [ unzip ];
+
+  unpackPhase = ''
+    mkdir www
+    unzip $src -d www
+  '';
+
+  # We use a sub-directory to accomodate other types of files that might
+  # be added in the future.
+  installPhase = ''
+    mkdir $out
+    cp -r www $out/
```

Comment:
We can't have $out/www.

Reply:
Ok, moved it to `$out/share/fluidd/htdocs`.